templates/search/catalogue_search.html

```diff
@@ -10,19 +10,16 @@
         {%  with type='Catalogue'%}
             {% include './blocks/search_results_hero.html' %}
         {% endwith  %}
-
+        <p class="search-results__explainer">Results for records held at The National Archives and other archives that match your search term.</p>
         {% include './blocks/catalogue_search_buckets.html' %}
         {% comment %} TODO make this text an editable field {% endcomment %}
-        {% if form.group.value == 'tna' %}
-            <p class="search-results__explainer--desktop">This page displays search results for records that are held by The National Archives, Kew.</p>
-            <p class="search-results__explainer--mobile">Results for records that are held by The National Archives, Kew.</p>
-        {% elif form.group.value == 'digitised' %}
-            <p class="search-results__explainer--desktop">This page displays search results for records that are available to download that are held by The National Archives, Kew.</p>
-            <p class="search-results__explainer--mobile">Records that are available to download that are held by The National Archives, Kew.</p>
-        {% elif form.group.value == 'nonTna' %}
-            <p class="search-results__explainer--desktop">This page displays search results for records held by other archives in the UK and beyond. None of the records shown here are held by The National Archives, Kew.</p>
-            <p class="search-results__explainer--mobile"> Results for records held by other archives in the UK and beyond. None here are held by The National Archives, Kew.</p>
-        {% endif %}
+        {% for bucket in buckets %}
+            {% if bucket.is_current and bucket.description %}
+                <p class="search-results__explainer">
+                    {{bucket.description}}
+                </p>
+            {% endif %}
+        {% endfor %}
```

Comment:
You can use the `BucketList`s `current`  property to get the current bucket without a loop :)

```suggestion
        {% if buckets.current.description %}
            <p class="search-results__explainer">
                {{ buckets.current.description }}
            </p>
        {% endif %}
```

Reply:
Thanks @ababic - I naively only tried the variable `bucket_list` - didn't realise it was called buckets!